docs/src/pages/docs/events.md

```diff
@@ -0,0 +1,200 @@
+# Events
+
+Events are an incredibly powerful feature in Anchor. Rather than polling or refreshing to
+find out if an account's state has changed, events are like callbacks that save space, compute,
+and enable asynchronous programming.
+
+The main downside to events are that they are base64 encoded, and therefore not human readable.
+Despite this tradeoff, the UI can easily decode them, and the user benefits from this 
+compact mode of logging.
+
+There are a couple of ways to use events, which vary somewhat:
+- to provide a return value
+- as a callback or interrupt for event-driven programming
+- as a compact and efficient means of data logging
+
+*Return values*: Here we use an event to immediately provide a return value, such as details related
+to a transaction. Since calls to certain programs only return a tx hash, this may not provide sufficient detail to your UI. An event here can provide a richer experience, by returning values about how a particular transaction was executed, for example.
+
+*As a callback*: you may want to monitor an event and be notified only once a transaction is confirmed.
+Instead of polling / querying the blockchain and looking for an account's values to update, we can build our client to only act when the program has emitted an event, saving on compute resources and fees.
+
+*Cheap storage*: since events are stored in base64, they can be an economical way to store data
+without the trouble of account creation and provisioning for rent-exemption, for example.
```

Comment:
```suggestion

## Event Use Cases

### Return Values
We can use an event to immediately provide a return value, such as details related
to a transaction. Since calls to certain programs only return a tx hash, this may not provide sufficient detail to your UI. An event here can provide a richer experience, by returning values about how a particular transaction was executed, for example.

### Callbacks
 You may want to monitor an event and be notified only once a transaction is confirmed.
Instead of polling / querying the blockchain and looking for an account's values to update, we can build our client to only act when the program has emitted an event, saving on compute resources and fees.

### Compact Storage 
Since events are stored in base64, they can be an economical way to store data
without the trouble of account creation and provisioning for rent-exemption, for example.
```

Reply:
![Screen Shot 2022-07-05 at 4 19 07 PM](https://user-images.githubusercontent.com/60185486/177418342-04758a63-037f-492c-a5ac-e71da2aabde9.png)
Here's a screenshot to show how the formatting looks.